AllJoyn/Platform/Bridge/BridgeCore/AllJoynHelper.cpp

```diff
@@ -1242,6 +1359,10 @@ QStatus AllJoynHelper::GetSignature(_In_ PropertyType propertyType, _Out_ std::s
         signature = "as";
         break;
 
+    case PropertyType::StringDictionary:
+        signature = "a{ss}";
```

Comment:
Reference [https://allseenalliance.org/docs/api/c/_msg_arg_8h.html#ab07552c43501e981dc8d026c3e0d2c45](https://allseenalliance.org/docs/api/c/_msg_arg_8h.html#ab07552c43501e981dc8d026c3e0d2c45)

---
In reply to: [67956680](https://github.com/tjaffri/msiot-samples/pull/7#discussion_r67956680) [](ancestors = 67956680,67949524,67949261)